packages/grid/package.json

```diff
@@ -49,6 +49,7 @@
     "@vaadin/component-base": "23.3.0-alpha0",
     "@vaadin/lit-renderer": "23.3.0-alpha0",
     "@vaadin/text-field": "23.3.0-alpha0",
+    "@vaadin/tooltip": "23.3.0-alpha0",
```

Comment:
No longer be needed once `tooltip-controller` is imported from `component-base`

Reply:
Removed.